app/services/ReviewService.scala

```diff
@@ -147,4 +115,56 @@ class ReviewService @Inject() (publishedService: PublishedService, repository: A
         }
     }
 
+  private def changeStatus(id: String, status: String, userId: String, reviewType: String): Future[RequestOutcome[Unit]] =
+    repository.changeStatus(id, status, userId) map {
+      case Left(Errors(DatabaseError :: Nil)) =>
+        logger.error(s"$reviewType - database error changing status")
+        Left(Errors(InternalServiceError))
+      case error@Left(Errors(NotFoundError :: Nil)) =>
+        logger.warn(s"$reviewType: Change Status: process $id was not found")
+        error
+      case Left(errors) =>
+        logger.error(s"$reviewType: changeStatus: for $id - $errors")
+        Left(errors)
+      case Right(_) => Right(())
+    }
+
+  private def getContentToUpdate(id: String, expectedStatus: String): Future[RequestOutcome[ApprovalProcess]] = repository.getById(id) map {
```

Comment:
I would probably use getApprovalProcessToUpdate here because you are retrieving an instance of ApprovalProcess.

Reply:
done, will be in next commit